wagtail/admin/locale/ca/LC_MESSAGES/django.po

```diff
@@ -695,12 +695,12 @@ msgid "No log entries found."
 msgstr "No s'han trobat entrades de registre."
 
 #, python-format
-msgid "No %(model_name)s match your query"
+msgid "No %(model_name)s match your query."
```

Comment:
Not sure how we feel about manually editing translation files, but I think we should try not to add extra work for translators. These are all in a separate commit d2da6b49be5439331d331e3791d3fc98b66f4f22 though, so if it's undesired I can drop it.

Reply:
Happy to keep this in, but we'll need to think about how to push this change to Transifex. I _think_ fetching translations, then rebasing/merging this PR on top, then `tx push` should do it...

Reply:
Okay, I'll do that when I merge (or leave it to you if you'd like to merge)